src/fetching/views/patchView.ts

```diff
@@ -0,0 +1,20 @@
+import APIError from 'utils/apiError';
+import { defaultFetch } from '..';
+import { ZetkinView } from 'types/zetkin';
+
+export default function patchView(orgId: string | number, viewId: string | number, fetch = defaultFetch) {
+    return async (view: Record<string, unknown>):Promise<ZetkinView> => {
+        const url = `/orgs/${orgId}/people/views/${viewId}`;
+        const res = await fetch(url, {
+            body: JSON.stringify(view),
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            method: 'PATCH',
+        });
+        if (!res.ok) {
+            throw new APIError('PATCH', url);
+        }
+        return await res.json();
```

Comment:
All Zetkin API requests that return something put it in to the property `data`. Most of our fetch handlers are like this
```js
const resBody = await res.json();
return resBody.data
```

Because of this the return type isn't correct, the return type presently would be `Promise<{ data: ZetkinView}>`

Even though you aren't using this handler in any case where you need to return this value, for consistency's sake, and for types' sakes, I would say it's best to return the `resBody.data`.